### ISSUE TITLE
Can I use jump\bastion host?

### ISSUE THREAD

**Issue by niko-lay** [spook/sshping#24]
Hello
Is there any ability to use jump\bastion in connection string? As `ssh -J jumpHost user@host` in general ssh

**Comment by spook** [comment]
No, but you could use something like `proxytunnel` to handle the jumpstation.

**Comment by spook** [comment]
@niko-lay Does the -F option to specify a custom config meet your needs for this?  Of do you still want a separate -J option?

**Comment by niko-lay** [comment]
@spook thank you for provided option, but looks like  my ssh config does not work with sshping
